rules-tests/Php81/Rector/FuncCall/NullToStrictStringFuncCallArgRector/Fixture/skip_after_is_string_on_object_call.php.inc

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Rector\Tests\Php81\Rector\FuncCall\NullToStrictStringFuncCallArgRector\Fixture;
+
+final class SkipAfterIsStringOnObjectCall
+{
+    public ?string $stringNull = 'foo';
```

Comment:
does this also work with a method instead of a property?

Reply:
should be work if native type defined, the `NullToStrictStringFuncCallArgRector` is to work with safe change, 

- with `@param string`, it means already string, with native string to be skipped to avoid too many change,
- with native type, it means already string as well.

Reply:
@OskarStark you can try on https://getrector.com/demo whether you find another case :)

Reply:
Thanks for the info